x-pack/plugins/cases/server/services/user_actions/helpers.ts

```diff
@@ -339,24 +326,14 @@ export const buildSubCaseUserActions = (args: {
 export const buildCaseUserActions = (args: {
   actionDate: string;
   actionBy: User;
-  originalCases: Array<SavedObject<ESCaseAttributes>>;
-  updatedCases: Array<SavedObjectsUpdateResponse<ESCaseAttributes>>;
+  originalCases: Array<SavedObject<CaseAttributes>>;
+  updatedCases: Array<SavedObjectsUpdateResponse<CaseAttributes>>;
 }): UserActionItem[] => {
-  const getField = (
-    so: Pick<SavedObjectsUpdateResponse<ESCaseAttributes>, 'attributes'>,
-    field: UserActionFieldType
-  ) => {
-    return field === 'connector' && so.attributes.connector
-      ? transformESConnectorToCaseConnector(so.attributes.connector)
```

Comment:
Since we no longer need to do the transformation, we no longer need the abstraction for `getField` both can just use `get(so, ['attributes', field])` directly as done above.